### PR TITLE
Add file icons

### PR DESCRIPTION
**Pull request by a-b-r-o-w-n**
Just adds file icons in tree view and tabs. [file-icons](https://atom.io/packages/file-icons)

### CODE REVIEW

**Comment by aaronjensen** [comment]
hm, tab space is somewhat at a premium... i don't mind it on the fuzzy finder, but not sure about making tabs wider when most of my files in a project are going to have the same icon...

**Comment by a-b-r-o-w-n** [comment]
@aaronjensen I agree. I added styles to hide those.

File: styles.less [hunk 1]
```diff
@@ -30,3 +30,7 @@ atom-text-editor {
 atom-text-editor::shadow .cursor {
   // border-color: red;
 }
+
+.tab .title:before {
```
Comment (aaronjensen) [inline styles.less]: Does this hide just the new icons or also the find results icon? For very different window types (where the icon itself will tell you what the window is) I kinda like it...

Comment (a-b-r-o-w-n) [inline styles.less]: Good question. I'll look into it.

Comment (a-b-r-o-w-n) [inline styles.less]: I have a patch open on the file-icons repo to support this config. We can hold off on merging until that gets updated.

**Comment by a-b-r-o-w-n** [comment]
I updated the package version and config so the file icons won't show up in tabs.

**Pull request merged**